benchmarks/lobsters/.rubocop.yml

```diff
@@ -0,0 +1,277 @@
+# Please do not 'fix' style issues without a compelling, metrics-driven
+# argument that a style change will materially improve cod equality.
+# https://github.com/lobsters/lobsters/pull/460
+
```

Comment:
We could probably remove the rubocop stuff?

Reply:
We definitely could. I had initially leaned in the direction of not removing stuff unless it was big or caused a problem -- e.g. .rspec file, CONTRIBUTING, random utilities in "extras", etc. But there's no reason that needs to be a hard-and-fast rule.

Reply:
Hm. Looks like some of the "extras" stuff is structural, though presumably not all of it.